wgsl/index.bs

```diff
@@ -8313,6 +8313,8 @@ Most built-in functions have tags of:
 Here is the list of exceptions:
 - All functions in [[#sync-builtin-functions]] have a [=call site tag=] of  [=CallSiteRequiredToBeuniform=].
 - All functions in [[#derivative-builtin-functions]], [[#texturesample]], [[#texturesamplebias]], and [[#texturesamplecompare]] have a [=call site tag=] of [=CallSiteRequiredToBeUniform=] and a [=function tag=] of [=ReturnValueMayBeNonUniform=].
+- All functions in [[#array-builtin-functions]] have [=parameter tags=] of
+    [=ParameterNoRestriction=] for each parameter.
```

Comment:
nitpick:  There is only one function here, and it's rather special.
Should this be phrased instead as just for that one function?  That's a little more conservative in case we add more array functions.  What if we add arrayLength for non-runtime-sized arrays.

Secondly, I think it's also true that arrayLength should have callsite tag CallSiteNoRestriction,  function tag of NoRestriction.